targets/unity-v2/source/Shared/Internal/PlayFabHttp/PlayFabUnityHttp.cs

```diff
@@ -27,15 +27,20 @@ public class PlayFabUnityHttp : IPlayFabTransportPlugin
 
         public void SimpleGetCall(string fullUrl, Action<byte[]> successCallback, Action<string> errorCallback)
         {
-            PlayFabHttp.instance.StartCoroutine(SimpleCallCoroutine(fullUrl, null, successCallback, errorCallback));
+            PlayFabHttp.instance.StartCoroutine(SimpleCallCoroutine("get", fullUrl, null, successCallback, errorCallback));
         }
 
-        public void SimplePutCall(string fullUrl, byte[] payload, Action successCallback, Action<string> errorCallback)
+        public void SimplePutCall(string fullUrl, byte[] payload, Action<byte[]> successCallback, Action<string> errorCallback)
         {
-            PlayFabHttp.instance.StartCoroutine(SimpleCallCoroutine(fullUrl, payload, (result) => { successCallback(); }, errorCallback));
+            PlayFabHttp.instance.StartCoroutine(SimpleCallCoroutine("put",fullUrl, payload, (result) => { successCallback(result); }, errorCallback));
```

Comment:
You no longer need to wrap successCallback in a lambda, its signature matches that parameter.
(fixing this will clean up a memory allocation, and avoid general lambda context issues)